doc/news/changes.h

```diff
@@ -38,6 +38,18 @@ inconvenience this causes.
 </p>
 
 <ol>
+ <li> Deprecated: The DataOutInterface::write_pvd_record() and
+ DataOutInterface::write_visit_record() functions were actually
```

Comment:
Well, actually I didn't deprecate it, nor intend to. It's much easier to call the function in `DataOutInterface` than to call the one in `DataOutBase` because it knows where to get all of the information.